package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@hashicorp/ember-cli-api-double",
-  "version": "3.1.2",
+  "version": "4.0.0",
```

Comment:
Could you revert this for the moment, we usually bump/tag and release all at the same time ty!

Reply:
will do - thx.

Reply:
Done